sympy/matrices/common.py

```diff
@@ -2019,6 +2019,71 @@ def replace(self, F, G, map=False, simultaneous=True, exact=None):
         return self.applyfunc(
             lambda x: x.replace(F, G, map=map, simultaneous=simultaneous, exact=exact))
 
+    def rot90(self, k=1):
+        """Rotates Matrix by 90 degrees
+
+        Parameters
+        ==========
+
+        k : int
+            Parameter k specifies the rotation
+            k is 0 when the Matrix is not rotated
+            k is +ve when the Matrix is rotated Clockwise
```

Comment:
please use `k > 0` instead of `is +ve`....this abbreviation will not be familiar to everyone. I was relieved to find that it is not used anywhere in SymPy.

Reply:
Ha ... thank you !! i'm just waiting for the checks to pass ... then I'll make all the necessary changes and commit them